api/src/main/java/app/services/AuthorService.java

```diff
@@ -63,4 +47,20 @@ public void delete(Long id) {
     public void deleteAll() {
         authorRepository.deleteAll();
     }
+
+    private AuthorInfo initAuthorInfo(Author author) {
+        return new AuthorInfo() {{
+            setName(author.getName());
+            setId(author.getId());
+            if (author.getBooks() != null)
+                setBooks(author.getBooks().stream().map((i) -> new BookInfo() {{
+                    setId(i.getId());
```

Comment:
Try not to create variables with one character like `i` or `n`. It smells bad

Reply:
Instead **`i`** insert **`bookItem`**?